src/Agent/NewRelic/Agent/Core/Agent.cs

```diff
@@ -427,30 +429,43 @@ public void RecordLogMessage(string frameworkName, object logEvent, Func<object,
             if (_configurationService.Configuration.LogEventCollectorEnabled)
             {
                 var logMessage = getLogMessage(logEvent);
```

Comment:
We only want to forward a log event without a message if it has a non-null exception.